iron-selectable.html

```diff
@@ -201,6 +202,18 @@
       this.selected = this._indexToValue(index);
     },
 
+    /**
+     * Force a synchronous update of the `items` property.
+     *
+     * NOTE: If you are using this method, you should probably consider an
+     * alternate approach. Synchronously querying for items is potentially
+     * slow for many use cases. The `items` property will update asynchronously
+     * on its own to reflect selectable items in the DOM.
```

Comment:
I think it'd be helpful to give guidance on how to know when the items will be ready to use.

Maybe mention the `iron-items-changed` event here as the recommended path?